README.md

```diff
@@ -32,6 +32,7 @@
     - [Train the model in the cloud](#train-the-model-in-the-cloud)
   - [📜 Test the newly trained model](#-test-the-newly-trained-model)
   - [🍴 Serve the newly trained model](#-serve-the-newly-trained-model)
+- [🚀 Install latest instructLab upgrades](#-upgrade-instructlab-to-latest-version)
```

Comment:
Hello, thank you for this great work! Can we change this title in the toc so its consistent with the section title? I like the "Upgrade instructLab to latest version"

Can we also move it so its after the "Chat with the new model (not optional this time)" section. I think that would be better so the workflow is still consistent.

Reply:
Also ensure the capitalization are consistent
instructlab  -> InstructLab

Reply:
Sure! Thanks for the review